src/features/profile/pages/ValidateEmailChange/ValidateEmailChange.tsx

```diff
@@ -8,7 +8,7 @@ import { useLogoutRoutine } from 'features/auth/helpers/useLogoutRoutine'
 import { navigateToHomeConfig } from 'features/navigation/helpers/navigateToHome'
 import { RootStackParamList, StepperOrigin } from 'features/navigation/RootNavigator/types'
 import { homeNavConfig } from 'features/navigation/TabBar/helpers'
-import { useEmailUpdateStatus } from 'features/profile/helpers/useEmailUpdateStatus'
+import { useEmailUpdateStatusV2 } from 'features/profile/helpers/useEmailUpdateStatusV2'
```

Comment:
je propose de renommer le hook V2 en pas V2, comme ça, ça change très peu de code

```suggestion
import { useEmailUpdateStatus } from 'features/profile/helpers/useEmailUpdateStatus'
```

Reply:
Bonne idée!